src/Components/Aspire.Azure.Messaging.ServiceBus/README.md

```diff
@@ -103,7 +103,7 @@ The .NET Aspire Azure Service Bus library supports [Microsoft.Extensions.Configu
 You can also pass the `Action<AzureMessagingServiceBusSettings> configureSettings` delegate to set up some or all the options inline, for example to configure the health check queue name from code:
 
 ```csharp
-builder.AddAzureServiceBus("sb", settings => settings.HealthCheckQueueName = "myQueue");
+builder.AddAzureServiceBusClient("sb", settings => settings.HealthCheckQueueName = "myQueue");
```

Comment:
There are a bunch more in this file that need to be fixed up. Search for `AddAzure`.

Reply:
Looks like there are a bunch more in others as well.